packages/core/src/shared/telemetry/spans.ts

```diff
@@ -252,7 +252,7 @@ export class TelemetryTracer extends TelemetryBase {
         const frame = this.switchContext(span)
 
         try {
-            const result = this.#context.run(frame, fn, span)
+            const result = this.#context.run(frame, fn, span as any)
```

Comment:
😬

Reply:
Out of curiosity which change made casting to any required?

Reply:
After bumping `@types/node` tsc gave this error... I don't get it.

```
npm ERR! src/shared/telemetry/spans.ts(255,57): error TS2345: Argument of type 'TelemetrySpan<MetricBase>' is not assignable to parameter of type 'Metric<MetricShapes[U]>'.
npm ERR!   Property 'run' is missing in type 'TelemetrySpan<MetricBase>' but required in type 'Metric<MetricShapes[U]>'.
```